internal/daysteps/daysteps.go

```diff
@@ -1,13 +1,55 @@
 package daysteps
 
+import (
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	pd "github.com/Yandex-Practicum/tracker/internal/personaldata"
+	se "github.com/Yandex-Practicum/tracker/internal/spentenergy"
+)
+
 type DaySteps struct {
-	// TODO: добавить поля
+	Steps    int
+	Duration time.Duration
+	pd.Personal
 }
 
+var errData = errors.New("incorrect data")
+
 func (ds *DaySteps) Parse(datastring string) (err error) {
-	// TODO: реализовать функцию
+	s := strings.Split(datastring, ",")
+	if len(s) != 2 {
+		return errData
+	}
+
+	if v, err := strconv.Atoi(s[0]); err != nil {
+		return err
```

Comment:
А тут можно и обернуть
```suggestion
		return fmt.Errorf("invalid duration: %w", err)
```